dockerutil/file.go

```diff
@@ -0,0 +1,32 @@
+package dockerutil
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+func Copy(src, dst string) (int64, error) {
```

Comment:
nitpick:

Maybe `CopyFile` as an alternative name?

Without digging into the implementation, it was unclear to me what we were copying.

Reply:
done